criu/proc_parse.c

```diff
@@ -1981,7 +1981,7 @@ static int parse_fdinfo_pid_s(int pid, int fd, int type, void *arg)
 				e->has_inode = true;
 				e->has_pos = true;
 			} else {
-				eventpoll_tfd_entry free unpacked(e, NULL);
+				eventpoll_tfd_entry_free_unpacked(e, NULL);
```

Comment:
All fixes to the ongoing changes has to be merged into commits where they have been introduced.
git rebase --interactive orign/criu-dev should help you to do that.